contract-addresses/mainnetAddresses.ts

```diff
@@ -506,6 +506,11 @@ const MainnetAddresses: MainnetAddresses = {
     address: '0xe61d11ec732d556A26fb863B192052BEa03eF8B5',
     category: AddressCategory.Oracle
   },
+  chainlinkLUSDOracle: {
```

Comment:
please put external addresses in the external category

Reply:
this didn't need to get added. removing